cumulus/variables.tf

```diff
@@ -459,3 +459,11 @@ variable "html_template_dir" {
     Leave this field blank to use default templates that are included with the lambda code zip file.
   EOF
 }
+
+variable "CIRRUS_CORE_VERSION" {
+  type    = string
+}
+
+variable "CIRRUS_DAAC_VERSION" {
+  type    = string
+}
```

Comment:
Are we sure that we want to use screaming snake case for these variables? Looks like the majority of our variables use snake case.

I would also suggest moving them to the top of the file rather than the bottom as they should appear in every module in cirrus, and that way they'll be easy to find in a consistent place. I would say they should always be close by to and in the same order with `DEPLOY_NAME` and `MATURITY`.

Reply:
If I am going to keep by `DEPLOY_NAME` and `MATURITY` I am going to keep the as screaming snake case.

Reply:
You haven't addressed the fact that the placement within the variables files is inconsistent across modules.

Reply:
I had, I just did push it up.

Reply:
I hit it with the `git push` instead of the `git push --force`